src/Domain/Configuration.php

```diff
@@ -382,6 +376,13 @@ private function validateRequirements(): Closure
         };
     }
 
+    private function validatePresetClass(): Closure
+    {
+        return static function ($value): bool {
```

Comment:
you could use php7.4 short notation (`fn`) here if you want  :wink:

Reply:
I could, but I don’t really like PHP’s short syntax. However, it’s your project and if you want me to use it I will.